tests/pytests/test_cursors.py

```diff
@@ -246,3 +246,47 @@ def testExceedCursorCapacity(env):
 def testExceedCursorCapacityBG():
     env = Env(moduleArgs='WORKER_THREADS 1 MT_MODE MT_MODE_FULL')
     testExceedCursorCapacity(env)
+
+def testCursorOnCoordinator(env):
+    SkipOnNonCluster(env)
+    env.expect('FT.CREATE idx SCHEMA n NUMERIC').ok()
+    conn = getConnectionByEnv(env)
+
+    # Verify that empty reply from some shard doesn't break the cursor
+    conn.execute_command('HSET', 0 ,'n', 0)
+    res, cursor = conn.execute_command('FT.AGGREGATE', 'idx', '*', 'LOAD', '*', 'WITHCURSOR', 'COUNT', 1)
+    env.assertEqual(res, [1, ['n', '0']])
+    env.expect(f'FT.CURSOR READ idx {cursor}').equal([[0], 0]) # empty reply from shard - 0 results and depleted cursor
```

Comment:
Maybe we can monitor the shards and see that they get the _ft.cursor read command only when we send the ft.cursor read?